introduction/features.rst

```diff
@@ -446,49 +446,49 @@ then occur every three superblock cycles (approximately once per
 
-+-------------+-----------+--------------+------------+
-| Adjustment  | Miner %   | Masternode % | Change (%) |
-+=============+===========+==============+============+
-| 0 (Current) | 50.0      | 50.0         | 0.00%      |
-+-------------+-----------+--------------+------------+
-| 1           | 48.7      | 51.3         | 1.30%      |
-+-------------+-----------+--------------+------------+
-| 2           | 47.4      | 52.6         | 1.30%      |
-+-------------+-----------+--------------+------------+
-| 3           | 46.7      | 53.3         | 0.70%      |
-+-------------+-----------+--------------+------------+
-| 4           | 46.0      | 54.0         | 0.70%      |
-+-------------+-----------+--------------+------------+
-| 5           | 45.4      | 54.6         | 0.60%      |
-+-------------+-----------+--------------+------------+
-| 6           | 44.8      | 55.2         | 0.60%      |
-+-------------+-----------+--------------+------------+
-| 7           | 44.3      | 55.7         | 0.50%      |
-+-------------+-----------+--------------+------------+
-| 8           | 43.8      | 56.2         | 0.50%      |
-+-------------+-----------+--------------+------------+
-| 9           | 43.3      | 56.7         | 0.50%      |
-+-------------+-----------+--------------+------------+
-| 10          | 42.8      | 57.2         | 0.50%      |
-+-------------+-----------+--------------+------------+
-| 11          | 42.3      | 57.7         | 0.50%      |
-+-------------+-----------+--------------+------------+
-| 12          | 41.8      | 58.2         | 0.50%      |
-+-------------+-----------+--------------+------------+
-| 13          | 41.5      | 58.5         | 0.30%      |
-+-------------+-----------+--------------+------------+
-| 14          | 41.2      | 58.8         | 0.30%      |
-+-------------+-----------+--------------+------------+
-| 15          | 40.9      | 59.1         | 0.30%      |
-+-------------+-----------+--------------+------------+
-| 16          | 40.6      | 59.4         | 0.30%      |
-+-------------+-----------+--------------+------------+
-| 17          | 40.3      | 59.7         | 0.30%      |
-+-------------+-----------+--------------+------------+
-| 18          | 40.1      | 59.9         | 0.20%      |
-+-------------+-----------+--------------+------------+
-| 19          | 40.0      | 60.0         | 0.10%      |
-+-------------+-----------+--------------+------------+
++-------------+-----------+--------------+------------+--------------------------+
+| Adjustment  | Miner %   | Masternode % | Change (%) | Reallocation Dates (UTC) |
++=============+===========+==============+============+--------------------------+
+| 0 (Current) | 50.0      | 50.0         | 0.00%      |    2020.08.26 10:40      |
++-------------+-----------+--------------+------------+--------------------------+
+| 1           | 48.7      | 51.3         | 1.30%      |    2020.09.25 18:06      |
```

Comment:
The date for this first adjustment to 51.3% would be [this line](https://docs.google.com/spreadsheets/d/1PQEjUx9QsfyHw4Lc2_LWbffb_OQx1rI54_84kUNyLeo/edit#gid=1451835240&range=A48:E48) (see column E)